systemtest/060-delete.bats

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bats
+#
+# Copy tests
+#
+
+load helpers
+
+function setup() {
+    standard_setup
+
+    enable_delete=true
+    start_registry --enable-delete=$enable_delete reg
```

Comment:
(Non-blocking nit: I don’t think the `enable_delete` variable here helps anything. `--enable-delete=true` would be shorter.)

Reply:
sure, I will delete useless variable, thanks.